### DISCUSSION TITLE
Should we remove `SablierV2OpenEnded` prefix from error names?

### DISCUSSION THREAD

**Discussion by smol-ninja** [sablier-labs/flow#51] [Ideas]
### Motivation
1. There is only one contract.
2. All errors are emitted from `SablierV2OpenEnded` contract.
3. Prefixing errors with `SablierV2OpenEnded` makes them longer to read.
4. In lockup contracts it makes sense to use prefix because it has more than one contract.

**Comment by andreivladbrg** [comment]
I will apply the same argument from here: https://github.com/sablier-labs/v2-open-ended/pull/49#discussion_r1582912704

It makes debugging easier and it improves the DX.

Also tagging @PaulRBerg as he is the one who introduces this method of writing custom errors.

**Comment by PaulRBerg** [comment]
Errors are there not for us internally, but for the external world. It makes most sense to keep using the contract name as a prefix:

https://twitter.com/PaulRBerg/status/1510584043028500492

> Prefixing errors with SablierV2OpenEnded makes them longer to read.

Feature, not bug

**Comment by smol-ninja** [comment]
Thank you both for your comments. I agree and we can close this.
